docs/docsite/rst/shared_snippets/installing_collections.txt

```diff
@@ -18,6 +18,29 @@ You can also directly use the tarball from your build:
 
    ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections
 
+You can build and install a collection from a local source directory. The ``ansible-galaxy`` utility builds the collection using the ``MANIFEST.json`` or ``galaxy.yml``
+metadata in the directory.
+
+.. code-block:: bash
+
+   ansible-galaxy collection install /path/to/collection -p ./collections
+
+You can also install multiple collections in a namespace directory.
+
+.. code-block:: text
+
+   ns
+   ├── collection1
```

Comment:
I think it's better to use the `ls -Fl` format and add trailing slashes to dirs
```suggestion
   ns/
   ├── collection1/
```

Reply:
thanks!